api/docker-compose.yml

```diff
@@ -49,9 +49,6 @@ services:
 
       # https://faimsproject.atlassian.net/wiki/spaces/FAIMS3/pages/151846960/FAIMS3+Conductor+build+run+environment+variables
 
-      # optional
-      - CLUSTER_ADMIN_GROUP_NAME
-
```

Comment:
This is not used anywhere in the app - I've moved this to data models in a new file. This will be important for the front end to be able to reason about what parts of the application the user should be able to see as we move functionality into the main app.